codeinterpreterapi/schema/file.py

```diff
@@ -38,7 +42,7 @@ def save(self, path: str):
     async def asave(self, path: str):
         await asyncio.to_thread(self.save, path)
 
-    def show_image(self):
+    def get_image(self):
```

Comment:
This is for showing the image not getting but you can add an additional get_image method in case you need this. The show_image method needs to stay.